app/frontend/src/styles/application/jobseekers/jobseeker.scss

```diff
@@ -1,4 +1,13 @@
 .jobseeker {
+  &.jobseekers_sessions_new {
+    .govuk-main-wrapper {
+      > .govuk-width-container {
+        margin-top: govuk-spacing(4);
+      }
+    }
+  }
+
+  &.publishers_identifications_new,
```

Comment:
this should really be here but the location/naming of controllers/templates look to need a look at. `jobseeker publishers_identifications_new` are the classes currently applied to the school sign in page as opposed to `jobseeker jobseekers_sessions_new` for teacher sign in page